[RFR] Rewrite sustainable open source page (#235)
pymedphys/pymedphys: docs/supporters/sustainable.rst

#######################
Sustainable Open Source

> #######################

Author: This is something I don't yet understand with reST, what's the difference here?

Reviewer: See the [Sphinx guide for Sphinx-based documentations](https://documentation-style-guide-sphinx.readthedocs.io/en/latest/style-guide.html#headings).

'###' is meant for doc titles, '===' denotes a H3 subheading. I didn't see a formatting change though, so I think Sphinx might be promoting the highest level subheading to title and scaling every subheading accordingly...

The Plan
========

There are a number of ways for PyMedPhys to reach a sustainable model while
keeping to PyMedPhys' vision of providing a freely available, open-source code
library. One way is to seek advertising revenue. This is something we have now
started to actively pursue, though we are very mindful of detracting from user
experience. Another way is to provide guaranteed support via contracts. We
expect to pursue this once we have put together a decent suite of high quality
applications.

At no extra cost, we will say "thank you" to our supporters via `the supporters
page <supporters.html>`_. We want to make it clear that we appreciate their
partnership with us!

We will also make all revenue public, open and transparent.

Advertising
"""""""""""

We would like to implement minimally-intrusive advertising. We will provide a
user-friendly and integrated way for companies who want to become supporters to
bid for ad placement.

All advertising will be based upon `viewable impressions
<https://www.thinkwithgoogle.com/feature/viewability/#/demo>`_
using `a transparent open source viewability library
<https://kahwee.github.io/viewability/timer.html>`_. We would specifically

> exclude charging for advertising to any user who is a maintainer of the

Author: ```suggestion
exclude charging for advertising when the advert is viewed by a maintainer of the
```

Sustainable Open Source
=======================

Project Maintenance
-------------------

> In its current form, at the time of writing, there are five contributors, two

Reviewer: ```suggestion
In its current form, at the time of writing, there are five contributors. Two
```
